show/bgp_common.py

```diff
@@ -356,14 +357,22 @@ def show_routes(args, namespace, display, verbose, ipver):
         arg_strg += str(arg) + " "
         if str(arg) == "json":
             found_json = 1
+        elif str(arg) == "tables":
+            found_tables = 1
         else:
             try:
                 filter_by_ip = ipaddress.ip_network(arg)
             except ValueError:
                 # Not ip address just ignore it
                 pass
-    if not found_json:
-        arg_strg += "json"
+    # Due to options such as "summary" and "tables" are not yet supported in multi-asic platform
+    # we will let FRR handle all the processing instead of handling it here for non multi-asic platform
+    if multi_asic.is_multi_asic():
+        if found_tables:
```

Comment:
how are you checking for "summary" option there?

Reply:
For Non-multi-asic platform the parameter checking is all done by BGP itself.
For Multi-ASIC platform it currently will fail due to the added "json" parm was requested to BGP and BGP will fail the request.  So there is no need to specifically check for this for now.
I will be adding the support for the Multi-ASIC platform for the "summary" option which at that time it will be inspected and handled accordingly.  Not part of this PR though...